src/components/Dashboard/index.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, { Component } from "react";
+
+import Tile from "components/Tile";
+
+import { Station } from "types";
+import { toCelsius } from "utils";
+
+import "./styles.css";
+
+type Props = {
+  station: Station
+};
+
+class Dashboard extends Component<Props> {
+  render() {
+    return (
+      <div className="Dashboard">
+        <Tile>
```

Comment:
Je trouve ca correct, mais en forçant un peu.. on pourrait dire que ton `<Title><h1></h1></Title>` se répète peu être un peu. Je lance des idées comme ca.. au cas où tu penses que ça serait mieux
- Faire un component one-liner qui prend une string en prop et qui render dans Title + h1
ou bien
- Au lieux d'aller chercher chaque valeur séparément, tu pourrais mettre tout dans un array de string, passer ca à un component qui loop à travers les strings et les renders dans un TItle + h1.

Mais pour vrai... c'est ben correct haha